packages/rke2-coredns/rke2-coredns.patch

```diff
@@ -330,6 +330,19 @@ diff -x '*.tgz' -x '*.lock' -uNr packages/rke2-coredns/charts-original/values.ya
    pullPolicy: IfNotPresent
  
  replicaCount: 1
+@@ -34,10 +34,10 @@
+     prometheus.io/port: "9153"
+ 
+ serviceAccount:
+-  create: false
++  create: true
+   # The name of the ServiceAccount to use
+   # If not set and create is true, a name is generated using the fullname template
+-  name:
++  name: coredns
```

Comment:
Is there any reason to hardcode the name instead of just letting the chart generate it? If we're hardcoding this here instead of letting the chart generate a name, it looks we also have to hardcode the name at https://github.com/rancher/rke2-charts/blob/main/charts/rke2-coredns/values.yaml#L49

For CIS, do we want to also enable PSP for CoreDNS here: https://github.com/rancher/rke2-charts/blob/main/charts/rke2-coredns/values.yaml#L46

Reply:
I opted to hardcode it with the assumption that the old name would be left behind and we'd start littering names on each upgrade.